frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -768,6 +781,365 @@ class SegmentsView extends React.Component<Props, State> {
     </>
   );
 
+  getToastForMissingLocations = (groupId: number) => {
+    const segmentsWithoutPosition = this.getSegmentsWithMissingLocation(groupId);
+    if (segmentsWithoutPosition.length > 0) {
+      console.log(`Segments with unknown positions: ${segmentsWithoutPosition}`);
+      return Toast.info(
+        <React.Fragment>
+          {segmentsWithoutPosition.length} meshes couldn't be loaded because the segment position is
+          unknown. The segment IDs were printed to the console.
+        </React.Fragment>,
+      );
+    }
+  };
+
+  getSetGroupColorMenuItem = (id: number): ItemType => {
+    const isEditingDisabled = !this.props.allowUpdate;
+    return {
+      key: "changeGroupColor",
+      disabled: isEditingDisabled,
+      label: (
+        <ChangeColorMenuItemContent
+          title="Change Group Color"
+          isDisabled={isEditingDisabled}
+          onSetColor={(color) => {
+            if (getVisibleSegmentationLayer == null) {
+              return;
+            }
+            this.setGroupColor(id, color);
+          }}
+          rgb={[0.5, 0.5, 0.5]}
```

Comment:
how about defaulting to the color of the first segment here? especially, if all segments already have the same color, it can be weird to default to gray here.

Reply:
right, good idea, thanks!